.travis.yml

```diff
@@ -85,19 +85,20 @@ jobs:
         - ./tools/travis/checkAndUploadLogs.sh standalone
       name: "Standalone Tests"
     - script:
+        - sed -i "s@pause-grace = 50 milliseconds@pause-grace = 10 seconds@g"  ./core/invoker/src/main/resources/application.conf
```

Comment:
This is to minimize too frequent status changes(`PAUSED` <-> `RUNNING`).
It looks the system cannot forward activations to runtime containers in time due to limited resources.